test-case/multiple-pause-resume.sh

```diff
@@ -135,6 +136,27 @@ expect {
         if { \$i > $repeat_count } { exit 0 }
         exp_continue
     }
+    -re "#.*MAX" {
+        if { \$maxcount > 10 } { puts "\rMAX Volume error!"; exit 1 }
+        puts "\rError: MAX volume detected (\$maxcount/10), Wait for 500ms"
+        incr maxcount
+        after 500
+        exp_continue
+    }
+    -re "*:*\n"|"Hardware PCM card*" {
+        puts "\rexpect ignore dump-hw-params"
+        after 10
+        exp_continue
+    }
+    timeout {
+        puts "\rError: timeout detected, exit 1"
```

Comment:
```suggestion
     default {
        puts "\rError: expect time out or aplay is gone, exit 1"
```

Reply:
> And even if aplay or arecord is gone, timeout can handle the case.

I don't think so. Try passing a wrong --flag to aplay and you will see EOF but no timeout. I noticed this in a different sof-test and the result was PASS (now fixed)